homeassistant/components/comfoconnect/fan.py

```diff
@@ -72,50 +77,23 @@ def _handle_update(self, value):
         self.schedule_update_ha_state()
 
     @property
-    def should_poll(self) -> bool:
-        """Do not poll."""
-        return False
-
-    @property
-    def unique_id(self):
-        """Return a unique_id for this entity."""
-        return self._ccb.unique_id
-
-    @property
-    def name(self):
-        """Return the name of the fan."""
-        return self._name
-
-    @property
-    def icon(self):
-        """Return the icon to use in the frontend."""
-        return "mdi:air-conditioner"
-
-    @property
-    def supported_features(self) -> int:
-        """Flag supported features."""
-        return SUPPORT_SET_SPEED
-
-    @property
-    def percentage(self) -> int | None:
+    def percentage(self):
         """Return the current speed percentage."""
         speed = self._ccb.data.get(SENSOR_FAN_SPEED_MODE)
         if speed is None:
             return None
         return ranged_value_to_percentage(SPEED_RANGE, speed)
 
     @property
-    def speed_count(self) -> int:
+    def speed_count(self):
```

Comment:
Why has the typing be removed here?

Reply:
My IDE (Pycharm) used the inherited types just fine when I got rid of them, but I got an error when the types in my file were not up to date since the parent definition has changed.

I thought this was a cleaner way since they were inherited anyway, but I'll gladly copy them from the parent class if you like that better.